src/Asset.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\CoreStandards\Asset.
+ */
+
+namespace Netzstrategen\CoreStandards;
+
+/**
+ * Administrative back-end functionality.
+ */
+class Asset {
+
+  /**
+   * Replaces default version to Git reference in script URL.
+   *
+   * @return string
+   *   The tag to print with the version updated.
+   */
+  public static function loader_tag($tag, $handle, $src) {
+    $wp_scripts = wp_scripts();
+    $default_version = $wp_scripts->default_version;
+
+    return str_replace('?ver=' . $default_version, '?ver=' . self::getGitCommitHash(), $tag);
+  }
+
+  /**
+   * Gets the reference of the last commit.
+   *
+   * Helper function to handle asset version with wp_enqueue_*
+   * functions to bust caches.
+   *
+   * @see https://developer.wordpress.org/reference/functions/wp_enqueue_script/
+   *
+   * @return string|null
+   *   The first 8 characters of the git reference.
+   */
+  public static function getGitCommitHash() {
+    $git_version = NULL;
```

Comment:
This value should be statically cached on the class to only execute it once per request.